boards/nucleo-f334/Makefile.include

```diff
@@ -0,0 +1,56 @@
+## the cpu to build for
+export CPU = stm32f3
+export CPU_MODEL = stm32f334r8
+
+#define the default port depending on the host OS
+OS := $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyACM0
+else ifeq ($(OS),Darwin)
+  PORT ?= $(shell ls -1 /dev/tty.SLAB_USBtoUART* | head -n 1)
```

Comment:
BTW: the UART pins are located on the little add-on board (labeled tx/rx) in the default configuration if I remember this correctly.